### PR TITLE
[neco] Add a new subcommand "cke weight" to store weights of each role.

### PR DESCRIPTION
**Pull request by mitsutaka**
The label "cke.cybozu.com/weight" in cke-template.yml is different on each data center.
etc/cke-template.yml should not have the label as flexible
configuration.

### CODE REVIEW

**Comment by mitsutaka** [comment]
TODO: merge this branch and release CKE https://github.com/cybozu-go/cke/tree/update-go-module if approved.

File: progs/cke/conf.go [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 import (
+	"context"
 	"fmt"
 	"io"
 
+	"github.com/cybozu-go/cke"
+	"github.com/cybozu-go/cke/sabakan"
 	"github.com/cybozu-go/neco"
+	"github.com/cybozu-go/neco/storage"
 	yaml "gopkg.in/yaml.v2"
```
Comment (ymmt2005) [inline progs/cke/conf.go]: `sigs.k8s.io/yaml`

File: progs/cke/conf_test.go [hunk 2]
```diff
@@ -37,3 +42,256 @@ func TestGenerateConf(t *testing.T) {
...
+	err = yaml.Unmarshal(out, actual)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if !cmp.Equal(expected, actual) {
+		t.Error("unexpected file content:", cmp.Diff(expected, actual))
+	}
+
+	err = st.PutCKEWeight(ctx, map[string]float64{
+		"foo": float64(100),
```
[the remark this answers is not in the document]
Comment (mitsutaka) [reply progs/cke/conf_test.go]: Right.
It tests non-existing role does not affect cke template.

**Comment by satoru-takeuchi** [comment]
@ymmt2005 My review is done. Please merge this PR after confirming
your change request is reflected. When I took a look at the newest version,
it's done.

**Pull request merged**